### ISSUE TITLE
assignment mismatch: 8 variables but css.GetMatchedStylesForNode(nodes[0].NodeID).Do returns 9 values

### ISSUE THREAD

**Issue by KaineDesign** [chromedp/chromedp#1126]
Similar to #1121

#### What versions are you running?

```console
$ go list -m github.com/chromedp/chromedp
github.com/chromedp/chromedp v0.8.2
$ google-chrome --version
Google Chrome 103.0.5060.134 
$ go version
go version go1.18.2 linux/amd64
```

#### What did you do? Include clear steps.

```go
go run main.go
```

#### What did you expect to see?
a running program :P

#### What did you see instead?
```console
# github.com/chromedp/chromedp
/root/go/pkg/mod/github.com/chromedp/chromedp@v0.8.2/query.go:1139:33: assignment mismatch: 8 variables but css.GetMatchedStylesForNode(nodes[0].NodeID).Do returns 9 values
```

**Comment by ZekeLu** [comment]
It seems that `chromedp.MatchedStyle` is rarely used. And it's rather easy to call `css.GetMatchedStylesForNode` directly in the user code. I'm going to remove it.

**Comment by ZekeLu** [comment]
I have taken a close look into this issue, and found that this issue is in fact a duplicate of #1121. 

I have a better way to handle it without removing `chromedp.MatchedStyle`. A PR was just sent. I will close this issue once the PR is merged.
